src/context/errors.ts

```diff
@@ -6,7 +6,9 @@ import { json } from './json'
  * Returns a list of GraphQL errors.
  */
 export const errors = (
-  errorsList: Partial<GraphQLError>[],
-): ResponseTransformer<{ errors: typeof errorsList }> => {
+  errorsList?: Partial<GraphQLError>[],
+): ResponseTransformer | ResponseTransformer<{ errors: typeof errorsList }> => {
```

Comment:
I believe the correct annotation would be 

```ts
: ResponseTransformer<{ errors: typeof errorsList | undefined }>
```

Would this produce a proper type checking when given `undefined` as the `errors` value?

Reply:
I think is exactly the same isn't it?

`typeof errorsList` references a value that can be undefined, therefore inherit its types right?

Reply:
Not really. `errorsList` is `Partial<GraphQLError>[]` and doesn't contain a nullable type (always demands an array). 

So `undefined` is an incompatible type with `errorsList`. To handle this, we can use a union type `| undefined`.